.travis.yml

```diff
@@ -78,6 +78,11 @@ script:
   - qiita pet webserver --no-build-docs start &
   - sleep 5
   - QIITA_PID=$!
+  - source deactivate
```

Comment:
source doesn't need to be changed to conda at this point?

Reply:
I guess no, cause if that was the case the builds will fail and they seem fine on [that line](https://travis-ci.org/biocore/qiita/jobs/511147785#L814).

Reply:
thought so too, but thought I'd ask.